### PR TITLE
Update Kasten generate policy by preset label ClusterPolicy

### PR DESCRIPTION
**Pull request by mattbator**
## Related Issue(s)

N/A

## Description

Filenames, policy/rule names, category, and keywords have been updated to reflect Kasten K10 rebranding to Veeam Kasten.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

**Review by JimBugwadia — APPROVED** [review]

File: kasten/kasten-generate-policy-by-preset-label/artifacthub-pkg.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+displayName: Generate Kasten Policy from Preset
+createdAt: "2023-05-07T00:00:00.000Z"
+description: >-
+  Generates a Kasten policy for a new namespace that includes a valid "dataprotection" label, if the policy does not already exist.
+
+  Use with "kasten-validate-ns-by-preset-label" policy to require "dataprotection" labeling on new namespaces.
+install: |-
+  ```shell
+  kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/kasten/kasten-generate-policy-by-preset-label/create-kasten-policies-clusterrole.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/kasten/kasten-generate-policy-by-preset-label/kasten-generate-policy-by-preset-label.yaml
```
Comment (chipzoller) [inline kasten/kasten-generate-policy-by-preset-label/artifacthub-pkg.yml:12]: @mattbator, I think this is causing a problem for us. This should only reference one command and the new location of the moved policy.

Comment (mattbator) [reply kasten/kasten-generate-policy-by-preset-label/artifacthub-pkg.yml:12]: Thanks for calling out @chipzoller!

It looks like I neglected to include `create-kasten-policies-clusterrole.yaml` as part of this PR. 

To clarify - is the issue the missing file or that the `install:` section of `artifacthub-pkg.yml` is explicitly limited to a single line? If the latter, is there a best practice for how recommended/required permissions are provided for generate policies?

For reference, I have an open PR for another generate policy (https://github.com/kyverno/policies/pull/998) with similar `install:` commands and properly includes the `create-kasten-policies-clusterrole.yaml` file - just want to know if I need to make changes to both. Thanks!

Comment (chipzoller) [reply kasten/kasten-generate-policy-by-preset-label/artifacthub-pkg.yml:12]: The current issue is with the `artifacthub-pkg.yml` file. The `kubectl apply` command is only for installation of the policy and should reference the "final" location of the policy after any changes in the PR are executed.

Comment (chipzoller) [reply kasten/kasten-generate-policy-by-preset-label/artifacthub-pkg.yml:12]: Sent fix in #1036

**Pull request merged**